src/y0/algorithm/falsification.py

```diff
@@ -0,0 +1,114 @@
+from ..util import CITests
```

Comment:
can we make `pgmpy` a requirement and just import the functions in CITests directly from there?

Reply:
I wanted to be conservative in the number of dependencies.  We really only need this one file (and it could be reduced down to one function from this file) at this time.

Reply:
Excellent, then we can just copy it (just double check the licensing is okay and we make a very specific note about it)

Reply:
Its under MIT license, which I copied to the top of the file and have note that says where it was originally copied from.

Reply:
Perfect. Thank you!